test/e2e/capi_test.go

```diff
@@ -164,20 +164,20 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 		})
 	})
 
-	Context("Should successfully set and use node drain timeout", func() {
-		capi_e2e.NodeDrainTimeoutSpec(context.TODO(), func() capi_e2e.NodeDrainTimeoutSpecInput {
-			return capi_e2e.NodeDrainTimeoutSpecInput{
-				E2EConfig:             e2eConfig,
-				ClusterctlConfigPath:  clusterctlConfigPath,
-				BootstrapClusterProxy: bootstrapClusterProxy,
-				ArtifactFolder:        artifactFolder,
-				SkipCleanup:           skipCleanup,
-				ControlPlaneWaiters: clusterctl.ControlPlaneWaiters{
-					WaitForControlPlaneInitialized: EnsureControlPlaneInitializedNoAddons,
-				},
-			}
-		})
-	})
+	// Context("Should successfully set and use node drain timeout", func() {
+	// 	capi_e2e.NodeDrainTimeoutSpec(context.TODO(), func() capi_e2e.NodeDrainTimeoutSpecInput {
+	// 		return capi_e2e.NodeDrainTimeoutSpecInput{
+	// 			E2EConfig:             e2eConfig,
+	// 			ClusterctlConfigPath:  clusterctlConfigPath,
+	// 			BootstrapClusterProxy: bootstrapClusterProxy,
+	// 			ArtifactFolder:        artifactFolder,
+	// 			SkipCleanup:           skipCleanup,
+	// 			ControlPlaneWaiters: clusterctl.ControlPlaneWaiters{
+	// 				WaitForControlPlaneInitialized: EnsureControlPlaneInitializedNoAddons,
+	// 			},
+	// 		}
+	// 	})
+	// })
```

Comment:
Non-blocker note: I am not sure if we created an issue as a tracker to uncomment this test later on?

Reply:
See #5392

Reply:
Thank you Matt! https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/5392